conference.js

```diff
@@ -117,6 +117,8 @@ import { setSharedVideoStatus } from './react/features/shared-video';
 import { isButtonEnabled } from './react/features/toolbox';
 import { endpointMessageReceived } from './react/features/subtitles';
 
+import { JitsiStreamBlurEffect } from './react/features/stream-effect';
```

Comment:
You need to start using the newly created bundle instead of importing it because of the problem with the size. This way the 1MB for the blur effect is just moved to jitsi-meet.

Reply:
Loading is not done via loadScript